code/game/area/roguetownareas.dm

```diff
@@ -8,6 +8,15 @@ GLOBAL_LIST_INIT(roguetown_areas_typecache, typecacheof(/area/rogue/indoors/town
 	has_gravity = STANDARD_GRAVITY
 	ambientsounds = null
 	dynamic_lighting = DYNAMIC_LIGHTING_FORCED
+	var/forest_area = FALSE // Defines forest areas
+
+/area/rogue/Entered(mob/living/carbon/human) // Applies the forestguard debuff if someone in the forest garrison enters an area with forest_area = FALSE
+	. = ..()
+	if(HAS_TRAIT(human, TRAIT_FORESTGUARD))
+		if(src.forest_area == FALSE)
```

Comment:
```suggestion
		if(forest_area == FALSE)
```
Check if the Z is Town Z (2-6) and not forest area, so only the town areas need to be changed